logging_metrics_performance/enterprise_logging/enterprise-logging-setup.sh

```diff
@@ -16,10 +16,8 @@ then
         exit 1
 fi
 
-if [ $# -eq 2 ]
-  then
-    :
```

Comment:
I've left this here as a placeholder for determining masterurl and masterpublicurl automatically, like:

MCFG=/etc/origin/master/master-config.yaml
MU=$(grep -i masterurl $MCFG | head -n 1 | awk -F': ' '{print $2}')
MPU=$(grep -i masterpublicurl $MCFG | head -n 1 | awk -F': ' '{print $2}')

and then the script would take just `cluster_size` ($1) and `kibana_hostname` ($2).